config/software/git.rb

```diff
@@ -26,20 +26,24 @@
 
 relative_path "git-#{version}"
 
-version "1.9.0" do
-  source md5: "0e00839539fc43cd2c350589744f254a"
+version "2.7.1" do
```

Comment:
That's cool, I'll do a follow up with a default_version change that will be easier to back out if it causes problems.